### PR TITLE
chore: bump capi back to v0.1.1

### LINKED ISSUE #207: Cluster API v0.1.1 causes kubectl for pivot to fail
/kind bug

**What steps did you take and what happened:**
The bump to Cluster API v0.1.1 (d8065d7) introduced the following failure in the clusterctl flow:
```
I0525 06:12:46.992274   27939 clusterclient.go:986] Waiting for Machine calico-0-controlplane-0 to become ready...
I0525 06:12:46.994769   27939 clusterdeployer.go:97] Updating bootstrap cluster object for cluster calico-0 in namespace "default" with control plane endpoint running on calico-0-controlplane-0
I0525 06:12:47.011509   27939 clusterdeployer.go:102] Creating target cluster
I0525 06:12:47.011543   27939 getkubeconfig.go:38] Getting target cluster kubeconfig.
I0525 06:12:47.017788   27939 getkubeconfig.go:59] Waiting for kubeconfig on calico-0-controlplane-0 to become ready...
I0525 06:12:47.023212   27939 applyaddons.go:25] Applying Addons
I0525 06:12:47.023246   27939 clusterclient.go:923] Waiting for kubectl apply...
I0525 06:12:54.546236   27939 clusterdeployer.go:120] Pivoting Cluster API stack to target cluster
I0525 06:12:54.546281   27939 pivot.go:67] Applying Cluster API Provider Components to Target Cluster
I0525 06:12:54.546304   27939 clusterclient.go:923] Waiting for kubectl apply...
W0525 06:13:23.367521   27939 clusterclient.go:939] Waiting for kubectl apply... unknown error couldn't kubectl apply, output: namespace/azure-provider-system created
namespace/cluster-api-system created
customresourcedefinition.apiextensions.k8s.io/azureclusterproviderspecs.azureprovider.k8s.io created
customresourcedefinition.apiextensions.k8s.io/azureclusterproviderstatuses.azureprovider.k8s.io created
customresourcedefinition.apiextensions.k8s.io/azuremachineproviderspecs.azureprovider.k8s.io created
customresourcedefinition.apiextensions.k8s.io/azuremachineproviderstatuses.azureprovider.k8s.io created
customresourcedefinition.apiextensions.k8s.io/clusters.cluster.k8s.io created
customresourcedefinition.apiextensions.k8s.io/machineclasses.cluster.k8s.io created
customresourcedefinition.apiextensions.k8s.io/machinedeployments.cluster.k8s.io created
customresourcedefinition.apiextensions.k8s.io/machines.cluster.k8s.io created
customresourcedefinition.apiextensions.k8s.io/machinesets.cluster.k8s.io created
clusterrole.rbac.authorization.k8s.io/azure-provider-manager-role created
clusterrole.rbac.authorization.k8s.io/cluster-api-manager-role created
clusterrolebinding.rbac.authorization.k8s.io/azure-provider-manager-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/cluster-api-manager-rolebinding created
service/azure-provider-controller-manager-service created
statefulset.apps/azure-provider-controller-manager created
statefulset.apps/cluster-api-controller-manager created
secret/manager-bootstrap-credentials created
Error from server: error when creating "STDIN": etcdserver: request timed out
: exit status 1
I0525 06:13:23.367637   27939 createbootstrapcluster.go:36] Cleaning up bootstrap cluster.
I0525 06:13:23.367660   27939 kind.go:69] Running: kind [delete cluster --name=clusterapi]
I0525 06:13:24.430690   27939 kind.go:72] Ran: kind [delete cluster --name=clusterapi] Output: Deleting cluster "clusterapi" ...
F0525 06:13:24.430767   27939 create_cluster.go:61] unable to pivot cluster api stack to target cluster: unable to apply cluster api controllers: couldn't kubectl apply, output: namespace/azure-provider-system created
namespace/cluster-api-system created
customresourcedefinition.apiextensions.k8s.io/azureclusterproviderspecs.azureprovider.k8s.io created
customresourcedefinition.apiextensions.k8s.io/azureclusterproviderstatuses.azureprovider.k8s.io created
customresourcedefinition.apiextensions.k8s.io/azuremachineproviderspecs.azureprovider.k8s.io created
customresourcedefinition.apiextensions.k8s.io/azuremachineproviderstatuses.azureprovider.k8s.io created
customresourcedefinition.apiextensions.k8s.io/clusters.cluster.k8s.io created
customresourcedefinition.apiextensions.k8s.io/machineclasses.cluster.k8s.io created
customresourcedefinition.apiextensions.k8s.io/machinedeployments.cluster.k8s.io created
customresourcedefinition.apiextensions.k8s.io/machines.cluster.k8s.io created
customresourcedefinition.apiextensions.k8s.io/machinesets.cluster.k8s.io created
clusterrole.rbac.authorization.k8s.io/azure-provider-manager-role created
clusterrole.rbac.authorization.k8s.io/cluster-api-manager-role created
clusterrolebinding.rbac.authorization.k8s.io/azure-provider-manager-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/cluster-api-manager-rolebinding created
service/azure-provider-controller-manager-service created
statefulset.apps/azure-provider-controller-manager created
statefulset.apps/cluster-api-controller-manager created
secret/manager-bootstrap-credentials created
Error from server: error when creating "STDIN": etcdserver: request timed out
: exit status 1
```

**Environment:**
- cluster-api-provider-azure version: d8065d7
- Kubernetes version: (use `kubectl version`): v1.14.2
- OS (e.g. from `/etc/os-release`): 
```
NAME="Ubuntu"
VERSION="18.04.2 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.2 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
```

### PR DESCRIPTION
**Pull request by alexeldeib**
**What this PR does / why we need it**:
Re-bumps capi back to v0.1.1

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #207 (no real changes)

**Special notes for your reviewer**:

_Please confirm that if this PR changes any image versions, then that's the sole change this PR makes._

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by alexeldeib** [comment]
tangential: why is k8s-ci-robot laughing at my PR?? :rofl:

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ PULL_POLICY ?= Always
 MANAGER_IMAGE_NAME ?= cluster-api-azure-controller
 
 # A release should define this with the next version after 0.1.0
-MANAGER_IMAGE_TAG ?= dev
+MANAGER_IMAGE_TAG ?= 0.1.1
```
Comment (justaugustus) [inline Makefile]: This variable refers to the version of the capz controller manager, not the [Cluster API](https://github.com/kubernetes-sigs/cluster-api/releases/tag/0.1.1) controller manager.
Revert this change when you have a chance.

Comment (alexeldeib) [reply Makefile]: My bad -- didn't mean to commit this.

**Comment by justaugustus** [comment]
This overall looks good, but I want to take an opportunity to spawn a cluster locally before approving it.
ref: #207

**Comment by alexeldeib** [comment]
For sure. I mean, it's basically what you already committed. If you get any additional details around a repro of what you saw, feel free to share and I can sink some time into debugging.

**Comment by justaugustus** [comment]
Yep, when I committed it, it introduced what I thought was a regression, so I want to take some time to poke at it before we do the revert dance again.

**Comment by justaugustus** [comment]
There may be something funky happening on my local, but I'm holding this in the meantime:
/hold

```
I0606 05:37:09.716054   23463 round_trippers.go:419] curl -k -v -XPOST  -H "Accept: application/json, */*" -H "Content-Type: application/json" -H "User-Agent: clusterctl/v0.0.0 (linux/amd64) kubernetes/$Format" 'https://<apiserver>:6443/apis/cluster.k8s.io/v1alpha1/namespaces/default/machines'
I0606 05:37:09.800728   23463 round_trippers.go:438] POST https://<apiserver>:6443/apis/cluster.k8s.io/v1alpha1/namespaces/default/machines 409 Conflict in 84 milliseconds
I0606 05:37:09.800900   23463 round_trippers.go:444] Response Headers:
I0606 05:37:09.800934   23463 round_trippers.go:447]     Content-Length: 278
I0606 05:37:09.800980   23463 round_trippers.go:447]     Date: Thu, 06 Jun 2019 12:37:09 GMT
I0606 05:37:09.801014   23463 round_trippers.go:447]     Content-Type: application/json
I0606 05:37:09.801134   23463 request.go:942] Response Body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"machines.cluster.k8s.io \"capi-v011-1-controlplane-0\" already exists","reason":"AlreadyExists","details":{"name":"capi-v011-1-controlplane-0","group":"cluster.k8s.io","kind":"machines"},"code":409}
I0606 05:37:09.801758   23463 createbootstrapcluster.go:36] Cleaning up bootstrap cluster.
I0606 05:37:09.801942   23463 kind.go:69] Running: kind [delete cluster --name=clusterapi]
I0606 05:37:10.975104   23463 kind.go:72] Ran: kind [delete cluster --name=clusterapi] Output: Deleting cluster "clusterapi" ...
F0606 05:37:10.975138   23463 create_cluster.go:61] unable to pivot cluster api stack to target cluster: unable to pivot cluster API objects: failed to move Machine default:capi-v011-1-controlplane-0: error copying Machine default/capi-v011-1-controlplane-0 to target cluster: error creating a machine object in namespace default: machines.cluster.k8s.io "capi-v011-1-controlplane-0" already exists
```

**Comment by alexeldeib** [comment]
Clarification: is this the same error you saw before? It looks different, wanted to confirm.

**Comment by alexeldeib** [comment]
Maybe you can share the cluster/machine config you are using?

**Comment by timmycarr** [comment]
Can confirm the same error message with a build and generic deploy from master... 
`I0606 22:49:21.753939    4144 round_trippers.go:444] Response Headers:
I0606 22:49:21.753979    4144 round_trippers.go:447]     Content-Type: application/json
I0606 22:49:21.754040    4144 round_trippers.go:447]     Content-Length: 286
I0606 22:49:21.754124    4144 round_trippers.go:447]     Date: Fri, 07 Jun 2019 03:49:21 GMT
I0606 22:49:21.754244    4144 request.go:942] Response Body: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"machines.cluster.k8s.io \"carrt-cluster-1-controlplane-0\" already exists","reason":"AlreadyExists","details":{"name":"carrt-cluster-1-controlplane-0","group":"cluster.k8s.io","kind":"machines"},"code":409}
I0606 22:49:21.754775    4144 createbootstrapcluster.go:36] Cleaning up bootstrap cluster.
I0606 22:49:21.754891    4144 kind.go:69] Running: kind [delete cluster --name=clusterapi]
I0606 22:49:23.712285    4144 kind.go:72] Ran: kind [delete cluster --name=clusterapi] Output: Deleting cluster "clusterapi" ...
F0606 22:49:23.712370    4144 create_cluster.go:61] unable to pivot cluster api stack to target cluster: unable to pivot cluster API objects: failed to move Machine default:carrt-cluster-1-controlplane-0: error copying Machine default/carrt-cluster-1-controlplane-0 to target cluster: error creating a machine object in namespace default: machines.cluster.k8s.io "carrt-cluster-1-controlplane-0" already exists`

File: Gopkg.toml [hunk 2]
```diff
@@ -52,7 +52,7 @@ required = [
 
 [[constraint]]
   name = "sigs.k8s.io/cluster-api"
-  version = "0.1.0"
+  version = "0.1.1"
```
Comment (justaugustus) [inline Gopkg.toml]:
```suggestion
  branch = "release-0.1"
```

**Comment by alexeldeib** [comment]
@justaugustus catching up — you merged, but didn’t change the version? Was the issue not related to the backport?

**Comment by justaugustus** [comment]
@alexeldeib -- opted to merge and iterate, since you were moving the ball forward. Will follow up with another PR later.

**Comment by alexeldeib** [comment]
Got it. I can test with that change tomorrow. I see CAPA had the same issue so seems likely to be the culprit. 

Currently in between legs of a looong trip home, will be settled in tonight and can poke over the weekend until it’s resolved.

**Comment by justaugustus** [comment]
No rush, @alexeldeib. Fix is here: #211
